Assets/Scripts/UI/HealthBar.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Collections;
```

Comment:
It seems like the HealthBar and ManaBar are going to be very similar, so it might be worth trying to extract the common functionality so that it is not repeated.